03_launch_mgmt_cluster.sh

```diff
@@ -256,7 +256,7 @@ EOF
 
     touch bmo.out.log
     touch bmo.err.log
-    kubectl scale deployment metal3-baremetal-operator -n metal3 --replicas=0
+    kubectl scale deployment baremetal-operator-controller-manager -n metal3 --replicas=0
```

Comment:
the namespace should be baremetal-operator-system

Reply:
we can fix that later, to get the CI back running

Reply:
I will put a patch to fix the namespace